devices/src/virtio/net.rs

```diff
@@ -503,7 +508,7 @@ impl EpollHandler for NetEpollHandler {
         &mut self,
         device_event: DeviceEventT,
         _: u32,
-        _: EpollHandlerPayload,
+        payload: EpollHandlerPayload,
```

Comment:
nit: I would check that the other events do not have a payload *payload == Empty) to enforce correctness when using this EpollHandler.

Reply:
Looking at it again, and thinking it in the context of the existing code, there is actually no real need or value for this extra check. It would also look ugly in the code. So I withdraw my objection 👍